### DISCUSSION TITLE
Welcome to chemtools Discussions!

### DISCUSSION THREAD

**Discussion by FarnazH** [theochem/chemtools#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by panchajaviera** [comment]
Hello!
I knew about this project from a talk that you give at QCMM22 and I tried to install it, but since Python 2.7 is no longer supported and many of the packages are outdated (and Horton is now being divided into several projects), I would like to know if there's going a to be a newer version of Chemtools, available for Python 3.10 soon or not.

**Reply by PaulWAyers** [reply]
Hi, I'm going to @FarnazH so that I'm sure she sees this. What I know is that there is a new version of ChemTools that works with Python 3, but I'm not sure if it's available for widespread use yet.

**Reply by leila-pujal** [reply]
Hi, @PaulWAyers and @panchajaviera. Yes, you are right @PaulWAyers, @FarnazH updated Chemtools to use python 3.7 together with Horton 2.3. Below I put the commands you should use in your terminal to install it. We will update the information/installation instructions soon. If you run the tests for Chemtools, you may get that some of them fail, but they are related to missing proatoms for computing effective oxidation state EOS calculation, which is the latest thing we added to the code and we are still working on it. For all the other tools implemented, all the tests should pass successfully. Let me know if you encounter any problems using the commands.

```
# create a Python 3 conda environment
conda create -n chemtools_py37 python=3.7
conda activate chemtools_py37

# install HORTON2.3
conda install  -c conda-forge -c theochem horton --yes
conda install nose
# run tests (you should get "OK (SKIP=1)")
nosetests -v horton

# install ChemTools
conda install -c conda-forge git-lfs
git clone https://github.com/theochem/chemtools.git 
cd chemtools
pip install -e .
# run test (all tests should pass successfully except ones related to EOS)
nosetests -v chemtools
```
